src/applications/check-in/day-of/pages/CheckIn/DisplayMultipleAppointments.jsx

```diff
@@ -48,15 +41,12 @@ const DisplayMultipleAppointments = props => {
 
   useEffect(
     () => {
-      const vaosOn = isUpdatedApptPresentationEnabled
-        ? 'VAOS-design'
-        : 'non-VAOS-design';
-      const slug = `check-in-viewed-appointment-list-${vaosOn}`;
+      const slug = `check-in-viewed-appointment-list-VAOS-design`;
```

Comment:
the problem with removing it is that it looses the history. I think once we land on a name, it just sticks so that we get historical data. I did ask Ben about a couple other ones to see if we wanted to keep them, like the page view ones and he said yes.